src/serac/infrastructure/initialize.cpp

```diff
@@ -27,6 +35,46 @@ std::pair<int, int> getMPIInfo(MPI_Comm comm)
   return {num_procs, rank};
 }
 
+#define INFO_BUFFER_SIZE 32767
+
+std::string getHostName()
```

Comment:
This is temporarily here until I move it to Axom and then Axom is updated

Reply:
I was about to complain about the gross string handling, but it makes sense if the plan is to upstream it.